### PR TITLE
adds ineligibility for perks

### PR DESCRIPTION
**Pull request by sarthaktexas**

### CODE REVIEW

**Comment by sarthaktexas** [comment]
fixes #4548

File: app/views/events/promotions.html.erb [hunk 1]
```diff
@@ -46,12 +46,16 @@
   </p>
 </div>
 
+<% if (@event.country_us?) %>
 <div class="card pb0 mb3" id="event-helper">
+<% else %>
+<div class="card pb0 mb3 muted" id="event-helper">
+<% end %>
```
Comment (sampoder) [inline app/views/events/promotions.html.erb]:
```suggestion
<div class="card pb0 mb3 <%= "muted" unless @event.country_us? %>" id="event-helper">
```
check this rq for me, but I think this is valid ERB and imo cleaner / more readable

File: app/views/events/promotions.html.erb [hunk 2]
```diff
@@ -111,12 +115,16 @@
   </div>
 <% end %>
 
+<% if (@event.country_us?) %>
 <div class="card pb0 mb3" id="pvsa">
+<% else %>
+<div class="card pb0 mb3 muted" id="pvsa">
+<% end %>
```
Comment (sampoder) [inline app/views/events/promotions.html.erb]: (if what I sent above works, it'd be nice to include it here as well)

File: app/views/events/promotions.html.erb (hunk 1 above)
Comment (sarthaktexas) [reply app/views/events/promotions.html.erb]: i literally googled this and it said this was not valid but i didn't fact check that SO post

Comment (sarthaktexas) [reply app/views/events/promotions.html.erb]: i agree this is cleaner so i'll check later if this works

Comment (sampoder) [reply app/views/events/promotions.html.erb]: i searched the bank codebase, we seem to do it a fair bit
eg.
```erb
<span class="<%= "muted " unless defined?(selectable) %>mb1 flex items-center">
```

¯\_(ツ)_/¯

**Review by sampoder — APPROVED** [review]

**Review by garyhtou — APPROVED** [review]

**Pull request merged**